interface/new/new_comprehensive_save.php

```diff
@@ -28,11 +28,9 @@
 // other interfaces can still read the data during this lock, however
 // sqlStatement("lock tables patient_data read");
 
-$result = sqlQuery("SELECT MAX(pid)+1 AS pid FROM patient_data");
```

Comment:
I have far fewer than 10000 records, however I manually updated my last row's PID to 10000. When I ran the query it gave back 10001. I can try with 10000 actual records, standby